apps/src/gamelab/Exporter.js

```diff
@@ -289,7 +290,25 @@ export default {
     return exportExpoPackagedFilesEjs({entries});
   },
 
-  async publishToExpo(appName, code, animationOpts) {
+  async generateExpoApk(snackId, config) {
+    const session = new SnackSession({
+      sessionId: `${getEnvironmentPrefix()}-${project.getCurrentId()}`,
+      name: `project-${project.getCurrentId()}`,
+      sdkVersion: '31.0.0',
+      snackId,
+      user: {
+        sessionSecret: config.expoSession || EXPO_SESSION_SECRET
+      }
+    });
```

Comment:
Can this duplicated code be shared somewhere, maybe StudioApp.js?

Reply:
We need to refactor `Exporter.js` between gamelab and applab. They are similar in many ways, but also different in some important ways.